Classes/Http/MeowInformationRequester.php

```diff
@@ -59,19 +61,10 @@ public function request(): string
                 'The request did not return JSON data'
             );
         }
-
-        try {
-            // Get the content as a string on a successful request
-            $content = $response->getBody()->getContents();
-            return json_decode($content, true, flags: JSON_THROW_ON_ERROR)['fact']
-                ?? throw new \RuntimeException('The response from the service is not as expected.', 1666413230);
-        } catch (\JsonException $e) {
-            $this->logger->error($e->getMessage());
-            throw new \RuntimeException(
-                'The service did not return a valid response. Try again later.',
-                1666413231,
-                $e
-            );
-        }
+        // Get the content as a string on a successful request
+        $content = $response->getBody()->getContents();
+        $result = json_decode($content, true, flags: JSON_THROW_ON_ERROR)['fact'];
```

Comment:
That won't work if fact is not returned in response (undefined array key), therefore the `?? throw`syntax here to cache it.

This should not happen (except the service is changing returned format).

Reply:
In my daily life I am dealing with services that change their formats and return unexpected stuff without notice all the time. Then the administrator or developer needs to give them a call and say "What the f* have you been doing now". So this information might be important. We cannot rely on a format.

Reply:
Oh, ok, the thing about the array key I somehow didn't see